packages/updatePrescriptionStatus/tests/testHandler.test.ts

```diff
@@ -73,6 +73,31 @@ describe("Integration tests for updatePrescriptionStatus handler", () => {
     expect(mockSend).toHaveBeenCalledWith(expect.objectContaining(expectedItems))
   })
 
+  it("when input field is absent in a single item request, expect DynamoDB item without RepeatNo field", async () => {
+    const body = generateBody()
+    const entryResource: any = body.entry?.[0]?.resource
+    if (entryResource?.input) {
+      delete entryResource.input
+    }
+
+    const event: APIGatewayProxyEvent = generateMockEvent(body)
+    const expectedItems = generateExpectedItems()
+    const transactItem: any = expectedItems.input?.TransactItems?.[0]?.Put?.Item
+    if (transactItem?.RepeatNo) {
+      delete transactItem.RepeatNo
+    }
+
+    mockTransact.mockReturnValue(expectedItems)
+
+    const response: APIGatewayProxyResult = await handler(event, {})
+
+    expect(response.statusCode).toEqual(201)
+    expect(JSON.parse(response.body)).toEqual(responseSingleItem)
+
+    expect(mockSend).toHaveBeenCalledTimes(1)
+    expect(mockSend).toHaveBeenCalledWith(expect.objectContaining(expectedItems))
```

Comment:
this is the same expect as in the test "when single item in request, expect a single item sent to DynamoDB" above this, so how do we know that one contains RepeatNo and one does not?

Reply:
Checks for `RepeateNo` value have been added:
`expect(expectedItems.input.TransactItems[0].Put.Item.RepeatNo).toEqual({S: 1})`
`expect(expectedItems.input.TransactItems[0].Put.Item.RepeatNo).toEqual(undefined)`